rules-reviewed/hibernate/hibernate-catchall.windup.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0"?>
+<ruleset xmlns="http://windup.jboss.org/schema/jboss-ruleset" id="hibernate-catchall" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
Well, that is the different view on it and I accept that as possible way. 

The thing is that migration rules in core rulesets should support/provide the recommended migration path not minimalistic path. Because It depends on migrated application and team focus I think.
There could be migration focused on just for hibernate and then it will fail with minimalistic approach, but I get what you are trying to say.